script/smt-data-export

```diff
@@ -39,7 +39,11 @@ function usage() {
 
 function smt_mysql() {
   local query="$1"
-  mysql -u $SMT_MYSQL_USER -p$SMT_MYSQL_PASSWORD -sN -e "$query" $SMT_MYSQL_DATABASE
+  if [ "$SMT_MYSQL_PASSWORD" == "" ]; then
+    mysql -u $SMT_MYSQL_USER -sN -e "$query" $SMT_MYSQL_DATABASE
+  else
+    mysql -u $SMT_MYSQL_USER -p$SMT_MYSQL_PASSWORD -sN -e "$query" $SMT_MYSQL_DATABASE
```

Comment:
`-p$SMT_MYSQL_PASSWORD` might be affected by the same escaping issue we've experienced in Yast (i.e., a password containing an ampersand would break it). Might be safer to use `--password='$SMT_MYSQL_PASSWORD'`, although a single quotation mark would also break it. :thinking:

Reply:
We've tested it and bash takes care of escaping by itself, so no changes required :+1: